pyaedt/edb.py

```diff
@@ -2437,6 +2437,10 @@ def hfss_setups(self):
         """
         return {i.name: i for i in self.setups if i.setup_type == "kHFSS"}
 
+        self.save_edb()
```

Comment:
@dcrawforAtAnsys I think the following 3 lines should not be there. Please remove them.